EnglishTranslation-master/QUEST_LV_0100.tsv

```diff
@@ -2975,7 +2975,7 @@ QUEST_LV_0100_20150317_002974	Notice /Destroy the corrupted stone lanterns!
 QUEST_LV_0100_20150317_002975	Corrupted Lantern (2)
 QUEST_LV_0100_20150317_002976	Notice /These are not the only corrupted lanterns. {nl} Find and destroy other corrupted lanterns. #5
 QUEST_LV_0100_20150317_002977	Disgrace
-QUEST_LV_0100_20150317_002978	Go to destroy royal spell suppressor
+QUEST_LV_0100_20150317_002978	Go to destroy the Royal Magic Control Device

```

Comment:
If this is an option you select from a quest, it should be: "I'll destroy the ..."